packages/geocoder/src/apis/otp/index.ts

```diff
@@ -0,0 +1,51 @@
+
+// eslint-disable-next-line prettier/prettier
+import type { AutocompleteQuery  } from "../../geocoders/types"
+
+type FetchArgs = {
+  url: string
+  query: string
+}
+
+type OTPGeocoderResponse = {
+  results: {
+    lat: number,
+    lng: number,
+    description: string,
+    id: string
+  }[]
+}
+
+
+function run({  query, url }: FetchArgs): Promise<OTPGeocoderResponse> {
+  // TODO: Support corners/osm nodes?
+  return fetch(`${url}/geocode?corners=false&query=${query}`)
+    .then(res => res.text())
+    .then(res => JSON.parse(`{"results": ${res}}`));
```

Comment:
It feels a little weird to construct a JSON object as a string. Why does the results need to be wrapped in an object under "results" anyway? It seems like in the `geocoders/otp` file you could just remove the `.results` part of accessing the data and then also remove this string.

Reply:
I wish... Unfortunately the response returned by OTP is invalid JSON so we have to do some processing to it. I tried to figure out how to get a "raw" array working without pelias-stitch throwing a fit, but will have another go...

Reply:
Oh, OTP is returning a straight up array? I see. That should be an easy fix in OTP, we should consider ticketing it in OTP upstream.

Reply:
Others rely on this api being as is

Reply:
I think we could just leave an issue for it then, at least